Controller/ContentManagement/EnvironmentController.php

```diff
@@ -502,12 +502,12 @@ public function addAction(Request $request)
 
     /**
      * Edit environement (name and color). It's not allowed to update the elasticsearch alias.
-     * @param unknown $id
+     * @param integer $id
```

Comment:
int

Reply:
I agree, but the whole project contains "integer" issues. This will be taken care off when passing to "php 7.2 strict typing" (I have 360 other issues for now on level 0)